src/key-value.spec.js

```diff
@@ -1,14 +1,84 @@
+import { mount, shallow } from 'enzyme';
+import React from 'react';
+import KeyValue from './key-value';
+
 describe('ReactKeyValue', () => {
-  it('should have the proper class name');
-  it('should set the default props');
-  it('should get the initial state from the props');
-  it('should handle adding new rows');
-  it('should handle when a key changes');
-  it('should handle when a value changes');
-  it('should handle removing a row');
-  it('should have a renderRows method');
-  it('should render the correct amount of keyItems set by props.rows');
-  it('should render the correct amount of valueItems set by props.rows');
-  it('should render an "Add new" button');
-  it('should render an "Add new" button with a custom method');
+
+  it('should have the proper class name', () => {
+    const $ = shallow(<KeyValue />);
+    $.hasClass('key-value').should.be.true();
+  });
+
+  it('should set the default props', () => {
+    const $ = mount(<KeyValue />);
+    $.props().rows.should.be.an.Array();
+    $.props().onChange.should.be.a.Function();
+  });
+  it('should get the initial state from the props', () => {
+    const $ = mount(<KeyValue />);
+    $.state().should.eql({ 'rows': [] });
```

Comment:
@ruffle1986 yes I remember we discussed this, I just left a note for myself in case I forget about it and would be trying to resolve this. Next time I'll mark it with #note or something